src/child/index.js

```diff
@@ -33,6 +34,8 @@ currentWindowService.ready(() => {
     const store = currentWindow.contentWindow.opener.store;
     const rootElement = document.getElementById('app');
 
+    startLayoutsService(store);
```

Comment:
Given both of these layout services take the store, and look at actions, could they be expressed as middleware and folded into the store creation?

Reply:
I am applying the same pattern as the other services which all get injected the store.
However those service are special: they deal with window layouts, which should not be part of the store.

Reply:
Fair enough!